### ISSUE TITLE
Warning: the function `eshell/pwd' is not known to be defined.

### ISSUE THREAD

**Issue by futurist** [knpatel401/filetree#12]
With Emacs 27.2 startup with warning message:

```
In end of data:
filetree.el:1603:1:Warning: the function `eshell/pwd' is not known to be
    defined.
```

MacOS 10.15.7, with Tramp mode.

**Comment by knpatel401** [comment]
Thanks for the heads up.  I'll fix this and a couple of other warnings in the next push.

-Ketan

**Comment by knpatel401** [comment]
Should be fixed in [61e425c8](https://github.com/knpatel401/filetree/commit/61e425c81acf0086d2a86a0613d186f28743df0e)

Thanks!
Ketan
